### PR TITLE
Feature: add default impl for RaftStorage::get_log_entries and try_get_log_entry

### PR DESCRIPTION
**Pull request by drmingdrmer**
##### Feature: add default impl for RaftStorage::get_log_entries and try_get_log_entry

---

### CODE REVIEW

File: openraft/src/defensive.rs [hunk 1]
```diff
@@ -343,3 +300,59 @@ where
         Ok(())
     }
 }
+
+pub fn check_range_matches_entries<D: AppData, RNG: RangeBounds<u64> + Debug + Send>(
+    range: RNG,
```
Comment (ariesdevil) [inline openraft/src/defensive.rs:305]: RNG is usually short for random number generator

**Review by lichuang — APPROVED** [review]

File: openraft/src/defensive.rs (hunk 1 above)
Comment (drmingdrmer) [reply openraft/src/defensive.rs:305]: I agree, but there is a type that may conflict with it: `std::ops::Range`...

Comment (ariesdevil) [reply openraft/src/defensive.rs:305]: How about just `RB`

Comment (drmingdrmer) [reply openraft/src/defensive.rs:305]: Nice~

**Pull request merged**